pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/tasks/action/CreateVirtualEnvAction.java

```diff
@@ -76,9 +76,12 @@ public void buildVenv(@Nullable Consumer<File> customize) {
         OutputStream outputStream = new ByteArrayOutputStream();
         ExecResult execResult = project.exec(execSpec -> {
             container.setOutputs(execSpec);
+            // For virtualenv >= 16.1
+            // execSpec.environment("PYTHONPATH", new File(packageDir, "src"));
+            execSpec.environment("PYTHONPATH", packageDir);
```

Comment:
Due to interactions with Python interpreter we avoid setting PYTHONPATH intentionally.
We don't need this.

Reply:
This is just for the execution of the `virtualenv` shell out, so its effect is pretty narrow.  It should also work better for virtualenv >= 16.1 by the previous line.   I think invocation by `-m virtualenv` is more typical than running the `.py` file.